lib/catalog/martyrology.ts

```diff
@@ -2765,6 +2997,17 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
         century: 1,
       },
     },
+    // src:
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marthe-le-bouteiller-vierge
+    // - https://en.wikipedia.org/wiki/Aim%C3%A9e-Ad%C3%A8le_Le_Bouteiller
+    marthe_aimee_adele_le_bouteiller_virgin: {
+      canonizationLevel: CanonizationLevels.Blessed,
+      dateOfBeatification: '1990-11-04',
+      name: 'Marthe-Aimée-Adèle Le Bouteiller',
```

Comment:
While I’ve found several websites with `Marthe`, papal/royal/religious names are usually localised.

And while in French multiple first names are usually hyphenated (and English _usually_ writes them the same way), I suggest to remove the hyphen between the religious and first names.

Moreover, I think that `le` in French names is written in lowercase. I could be wrong, but even the [French Wikipedia](https://fr.wikipedia.org/wiki/Marthe_Le_Bouteiller#:~:text=Marthe%20le%20Bouteiller%20\(Percy) writes it that way unless it is used as an article/section title.

```suggestion
      name: 'Martha Aimée-Adèle le Bouteiller',
```